.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,16 +1,18 @@
 ---
-name: Bug Report
-about: Create a report to help us improve
+name: Bug Report(.Net SDK)
+about: Create a report to help us improve AI .NET SDK
 labels: bug
 ---
 
-**Describe your environment.** Describe any aspect of your environment relevant to the problem:
+**Is this a bug in .NET SDK?**
```

Comment:
No need for this part.

Reply:
I think since AI home is retired, there's no other place for users to get a consolidated view of issue path for all SDKs. This would ideally be the right place for us to re-direct before they make a mistake

Reply:
I think the consolidated view is the official document on docs.microsoft.com.